lib/bom.js

```diff
@@ -1350,7 +1350,7 @@ declare class WaveShaperNode extends AudioNode {
 
 // this part of spec is not finished yet, apparently
 // https://stackoverflow.com/questions/35296664/can-fetch-get-object-as-headers
-type HeadersInit = Headers | { [key: string]: string, ... };
+type HeadersInit = Headers | string[][] | { [key: string]: string, ... };
```

Comment:
changing this to `Array<[string, string]>` per

<img width="618" alt="Screen Shot 2020-05-14 at 9 44 04 AM" src="https://user-images.githubusercontent.com/3012/81941947-7dcf7580-95c7-11ea-94dc-8cdf5b96e196.png">